lib/msal-browser/docs/caching.md

```diff
@@ -1,3 +1,78 @@
-## Cache Storage
+# Caching in MSAL
 
-We offer two methods of storage for Msal, `localStorage` and `sessionStorage`.  Our recommendation is to use `sessionStorage` because it is more secure in storing tokens that are acquired by your users, but `localStorage` will give you Single Sign On accross tabs and user sessions.  We encourage you to explore the options and make the best decision for your application.
+When MSAL acquires a token, it caches it for future usage. MSAL manages token lifetimes and refreshing for you. The following MSAL APIs will perform one or more cache lookups to acquire tokens or retrieve accounts:
```

Comment:
Technically, all of our APIs perform cache lookups but only `acquireTokenSilent` returns tokens from the cache. IMO this section should only mention `acquireTokenSilent`

Reply:
Ah, very well, reverted this